core/modules/css/SConscript

```diff
@@ -8,20 +8,15 @@ Import('standardModule')
 pythonInc = env['PYTHON_INC_DIR']
 cppPath = env['CPPPATH'] + [pythonInc]
 
-# Use special C++ file suffix to avoid mix with standardModule()
-# Python module is generated in python/ directory for standardModule()
-swigged = env.SharedObject(source="cssLib.i",
-                           SWIGFLAGS=['-python', '-c++', '-Iinclude'],
-                           SWIGPATH=['..', pythonInc],
-                           CPPPATH=cppPath,
-                           SWIGCXXFILESUFFIX='_wrap.c++',
-                           SWIGOUTDIR=env.Dir("python"))
+# pybind11 Python wrapper goes to a separate library, cc file is also
+# excluded from standardModule()
+# pybind11 makes too many warnings when built with -pedantic
```

Comment:
Would be useful to know what the warnings are. Perhaps something we can get fixed upstream?